### PR TITLE
WIP Renaming refactor on short feature name and full feature name

### PR DESCRIPTION
**Pull request by wfxr**
`FeatureFullName` -> `FeatureName` (used in most public apis)
`FeatureName` -> `ShortFeatureName` (used in private apis)

### CODE REVIEW

File: internal/database/metadata/informer/feature.go [hunk 1]
```diff
@@ -34,9 +34,9 @@ func (c *FeatureCache) List(opt metadata.ListFeatureOpt) types.FeatureList {
 	}
 
 	// filter names
-	if opt.FeatureFullNames != nil {
+	if opt.FeatureNames != nil {
 		var tmp types.FeatureList
-		for _, fullName := range *opt.FeatureFullNames {
+		for _, fullName := range *opt.FeatureNames {
```
Comment (lianxmfor) [inline internal/database/metadata/informer/feature.go:39]: Does it seem more consistent to use the name here? When for-loop shortNames, use shortName

Comment (wfxr) [reply internal/database/metadata/informer/feature.go:39]: @lianxmfor make sense

File: pkg/oomstore/online_query_test.go [hunk 2]
```diff
@@ -134,8 +134,8 @@ func TestOnlineMultiGet(t *testing.T) {
 		{
 			description: "inconsistent entity type, fail",
 			opt: types.OnlineMultiGetOpt{
-				FeatureFullNames: inconsistentFeatures.FullNames(),
-				EntityKeys:       []string{"1234", "1235"},
+				FeatureNames: inconsistentFeatures.FullNames(),
```
Comment (lianxmfor) [inline pkg/oomstore/online_query_test.go:137]: here use `Names` to replace `FullNames`

File: pkg/oomstore/push.go [hunk 3]
```diff
@@ -20,12 +20,12 @@ func (s *OomStore) Push(ctx context.Context, opt types.PushOpt) error {
...
 	group := features[0].Group
-	if !stringSliceEqual(features.Names(), opt.FeatureNames) {
-		return errdefs.Errorf("FeatureNames %v does not match with group's features %v", opt.FeatureNames, features.Names())
+	if !stringSliceEqual(features.Names(), opt.ShortFeatureNames) {
+		return errdefs.Errorf("ShortFeatureNames %v does not match with group's features %v", opt.ShortFeatureNames, features.Names())
 	}
 	values := make([]interface{}, 0, len(features))
 	for _, f := range features {
-		for i, name := range opt.FeatureNames {
+		for i, name := range opt.ShortFeatureNames {
```
Comment (lianxmfor) [inline pkg/oomstore/push.go:28]: use `short` replace `name`

File: pkg/oomstore/types/options.go [hunk 4]
```diff
@@ -1,21 +1,22 @@
 package types
 
 type CreateFeatureOpt struct {
-	FeatureName string
+	// Feature name without prefix
+	Name        string
```
Comment (lianxmfor) [inline pkg/oomstore/types/options.go:5]: use `shortName` repalce `Name`

File: pkg/oomstore/export.go [hunk 5]
```diff
@@ -34,17 +34,17 @@ func (s *OomStore) ChannelExportBatch(ctx context.Context, opt types.ChannelExpo
...
-	if len(opt.FeatureNames) == 0 {
+	if len(opt.ShortFeatureNames) == 0 {
 		features, err = s.ListFeature(ctx, types.ListFeatureOpt{
 			GroupName: &revision.Group.Name,
 		})
 	} else {
-		fullNames := make([]string, 0, len(opt.FeatureNames))
-		for _, name := range opt.FeatureNames {
+		fullNames := make([]string, 0, len(opt.ShortFeatureNames))
+		for _, name := range opt.ShortFeatureNames {
```
Comment (lianxmfor) [inline pkg/oomstore/export.go:43]: use `shortName` to replace `Name`

**Review by lianxmfor — APPROVED** [review]
lgtm!

**Pull request merged**